src/commands/bash_hook.rs

```diff
@@ -197,6 +230,34 @@ pub fn execute(matches: &ArgMatches) -> Result<()> {
         )
         .replace("___SENTRY_LOG_FILE___", &log.display().to_string());
 
+    let mut hook_tags = "".to_string();
```

Comment:
This will break for multiple tags, as it'll be concatenated to `-t foo-t bar-tbaz`